core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -96,7 +96,7 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
 
   val startTime = System.currentTimeMillis()
 
-  private val stopped: AtomicBoolean = new AtomicBoolean(false)
+  private[spark] val stopped: AtomicBoolean = new AtomicBoolean(false)
```

Comment:
@tedyu Since it is `private[spark]`, I do not think we need to mark it as a developer api.